app/views/layouts/application.html.erb

```diff
@@ -5,12 +5,24 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
- [ ] Please try to fix this to provide the login page.

- 
![Capture](https://user-images.githubusercontent.com/63972670/197084053-a54b0de0-dcee-447a-9e28-79288ff304ee.JPG)

Reply:
You need to initialize the build process by running `./bin/dev` after `bundle install`